<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarSpaceDialog</class>
 <widget class="QDialog" name="CarSpaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pb_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_6">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>110</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_7">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_8">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>110</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_9">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>110</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_10">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>110</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>选择车位</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>11</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_12">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_13">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>13</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_14">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>14</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_15">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_16">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>250</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>16</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_17">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>250</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>17</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_18">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>250</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>18</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_19">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>250</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>19</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_20">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>250</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>781</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>781</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>781</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>20</width>
     <height>281</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>20</width>
     <height>281</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>20</width>
     <height>281</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>20</width>
     <height>281</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>20</width>
     <height>281</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>20</width>
     <height>281</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>20</width>
     <height>281</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_11">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>100</y>
     <width>20</width>
     <height>281</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_12">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>100</y>
     <width>20</width>
     <height>281</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
